dp/generate_climos.py

```diff
@@ -736,8 +736,13 @@ def update_generate_climos_history(netCDF_file, time_cdo_format, position=0):
         arguments_list = sys.argv
         arguments_list[0] = ": generate_climos"
         command = " ".join(sys.argv)
-
-        hist_line = time_cdo_format + ": start" + command
+        
+        # update_generate_climos_history for the start has to be called after the first or latter cdo command
+        # update_generate_climos_history for the end has to be called after the last cdo command
+        if position != 0:
+            hist_line = time_cdo_format + ": start" + command
+        else:
+            hist_line = time_cdo_format + ": end" + command
```

Comment:
Interesting. For the present usage, the inference of start/end from `position` is correct, in that it works. But it's not *necessarily* so; one can imagine (in a vague way at least) that there could be other desired use cases. In that case it would be worth adding an explicit boolean begin/end parameter to this method. For now, I suggest you just document this inference explicitly, in the docstring at the head of this function. Your example implies this, but it would be good to state it explicitly.